package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -1829,6 +1829,17 @@ void hostapd_ubus_notify_beacon_report(
 	blobmsg_add_u16(&b, "antenna-id", rep->antenna_id);
 	blobmsg_add_u16(&b, "parent-tsf", rep->parent_tsf);
 
+	if (rep_mode == MEASUREMENT_REPORT_MODE_ACCEPT) {
+		blobmsg_add_u8(&b, "accepted", 1);
+	} else {
+		blobmsg_add_u8(&b, "accepted", 0);
+		void *reject = blobmsg_open_table(&b, "reject-reasons");
```

Comment:
In short... pls don't add a table and just provide the raw value here... we can make the change later and add reason (if needed)

But currently i think it would be better to just provide the raw value and the tool parse it.
(or even do both a provide ALSO the raw value)

Reply:
Changed it.